Provider/provider_attribute_type.md

```diff
@@ -0,0 +1,117 @@
+# Provider Attribute Type
+
+## Overview
+
+* If you wish to record extra information about providers, you can create Provider Attributes and assign them to Provider Types.
+
+## Available operations.
+
+1. [List provider attribute types](#list-provider-attribute-types)
+2. [Create a provider attribute type](#create-a-provider-attribute-type)
+3. [Update a provider attribute type](#update-a-provider-attribute-type)
+4. [Delete a provider attribute type](#delete-a-provider-attribute-type)
+
+
+### List provider attribute types
+
+* #### List all non-retired provider attribute types.
+
+    Quickly filter provider attribute types with a given search query. Returns a `404 Not Found` status if the provider attribute type not exists.
+     If the user not logged in to  perform this action, a `401 Unauthorized` status returned.
+
+    ##### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *q* | `Search Query` | Display Name of provider attribute type.
+
+    ```console
+    GET /providerattributetype?q="Search Query"
+     ```
+
+* #### List provider attribute type by UUID.
+
+    Retrieve a provider attribute type by its UUID. Returns a `404 Not Found` status if the provider attribute type not exists. If the
+    user not logged in to  perform this action, a `401 Unauthorized` status returned.
+
+    ```console
+    GET /providerattributetype/:target_provider_attribute_type_uuid
+    ```
+
+### Create a provider attribute type
+
+* To Create a provider attribute type you need to specify below attributes in the request body.If the user not logged in to perform this action,
+ a `401 Unauthorized` status returned.
+
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *name* | `String` | Name of the provider attribute type (Required)
+    *description* | `String` | Description (Required)
+    *datatypeClassname* | `CustomDataType Resource` | Data type for the attribute type resource.OpenMRS provides **Custom data type resource** which gives flexibility to select the data type accordingly (Required)
+    *minOccurs* | `Number` | Minimum number of times this value can be specified for a single provider.Use `0` or `1` as the default value (Required)
+    *maxOccurs* | `Number` | Maximum number of times this value can be specified for a single provider (e.g., use 1 to prevent an attribute from being added to a provider multiple times)
+    *preferredHandlerClassname* | `Handler` | Handler sub resource for the Custom Data Type used.Can optionally define a specific handler class want to use (otherwise the framework will choose the best handler for the chosen datatype).To find which handlers to use for the Custom DataType please refer here
+    *datatypeConfig* | `String` | Allow the data type have any name and config it wants/needs.
```

Comment:
The description isn't clear – i.e., it doesn't clarify what a `dataTypeConfig` is or should contain.